models/gc_spider_baseline/utils/SpiderBaselineRunner.py

```diff
@@ -0,0 +1,194 @@
+"""
+-------------------------------------------------------------
+Mhub / DIAG - Run Module for the GC Spider baseline Algorithm
+-------------------------------------------------------------
+
+-------------------------------------------------------------
+Author: Sil van de Leemput
+Email:  sil.vandeleemput@radboudumc.nl
+-------------------------------------------------------------
+"""
+from pathlib import Path
+from typing import Dict
+import shutil
+import os
+import json
+
+import SimpleITK
+import numpy as np
+
+from mhubio.core import Instance, InstanceData, IO, Module, CT, MR, Meta
+
+# The GC Spider baseline algorithm is imported as a self-contained algorithm class with an execute method
+from TestSpine import Module as SpiderAlgorithm
+
+
+SPIDER_DATA_DIR = Path(os.environ["VERSEG_BASEDIR"])
+SPIDER_INTERNAL_DATASET_PATH = SPIDER_DATA_DIR / "datasets" / "spider_input"
+SPIDER_EXPERIMENT_DIR = SPIDER_DATA_DIR / "experiments" / "SPIDER-Baseline"
+SPIDER_INTERNAL_OUTPUT_DIR = SPIDER_EXPERIMENT_DIR / "results" / "spider_input"
+SPIDER_EXPERIMENT_ARGUMENTS_FILE = SPIDER_EXPERIMENT_DIR / "arguments.json"
+SPIDER_INTERNAL_DATASET_META_FILE = SPIDER_INTERNAL_DATASET_PATH / "metadata.json"
+
+
+@IO.Config('traversal_direction_up', bool, True, the="Direction to traverse the image")
```

Comment:
This parameter only matters when the input images are not presented in the expected orientation, i.e. the bottom of the image is expected to start with the lumbar spine, and move upwards to the cervical spine. If this is in the reverse order, you can set this parameter to False to try and match the data instead of flipping the imaging data. So yes the algorithm will produce a different output and we have added a better explanation here.